test/test_issue801

```diff
@@ -0,0 +1,23 @@
+  """
+Issue 715 - path query chaining issue
+Some incorrect matches were found when using oneOrMore ('+') and
+zeroOrMore ('*') property paths and specifying neither the
+subject or the object.
```

Comment:
This comment looks out of date.

Reply:
@darrengarvey I have updated the comment